Doc: Migrate terminology to emphasize pipeline for better seo (#14032)
elastic/logstash: docs/static/ls-to-cloud.asciidoc

[[connecting-to-cloud]]
=== Sending data to Elastic Cloud (hosted Elasticsearch Service)

Our hosted {ess} on https://cloud.elastic.co/[Elastic Cloud] simplifies safe, secure communication between {ls} and {es}. 

> When you configure the elasticsearch output plugin to use <<plugins-outputs-elasticsearch-cloud_id,`cloud_id`>> with either the <<plugins-outputs-elasticsearch-cloud_auth,`cloud_auth` option>> or the <<plugins-outputs-elasticsearch-api_key,`api_key` option>>, no additional SSL configuration steps are needed.

Reviewer: ```suggestion
When you configure the Elasticsearch output plugin to use <<plugins-outputs-elasticsearch-cloud_id,`cloud_id`>> with either the <<plugins-outputs-elasticsearch-cloud_auth,`cloud_auth` option>> or the <<plugins-outputs-elasticsearch-api_key,`api_key` option>>, no additional SSL configuration is needed.
```
